### DISCUSSION TITLE
抛开事实不谈，你们这群买房的难道一点错都没有吗？

### DISCUSSION THREAD

**Discussion by KisaraKira** [WeNeedHome/SummaryOfLoanSuspension#123] [General]
理性辱骂，严禁讨论
（手动狗头

**Comment by zhenwolong** [comment]
钓就硬钓

**Comment by saurylip** [comment]
具体说说买房的是犯了什么错？

**Reply by Wesleylr** [reply]
他都抛开事实不谈了，嘲讽一下而已

**Reply by luzhanmin** [reply]
大概错在买了房

**Reply by xiazhibin** [reply]
错在买房

**Reply by lzwbzd** [reply]
错在没有抛开事实

**Reply by BobbyYang9** [reply]
错在有人觉得你犯了错

**Reply by Panda-Hero** [reply]
投错胎了？

**Reply by imhappyok** [reply]
错在你没有躺平

**Reply by Yuesd** [reply]
所以不用住房了，直接租房

**Comment by wengxianxun** [comment]
谁的责任？

**Reply by LeeeeeeM** [reply]
美国是搬起石头砸自己的脚

**Reply by CharAznable98** [reply]
责任全在美方

**Reply by qq912276337** [reply]
美国黄石公园的石头都快被搬完了

**Reply by Godwuqiang** [reply]
> 美国黄石公园的石头都快被搬完了

那就去旧金山搬

**Reply by ppwq2022** [reply]
美国以此为荣:能打败美国的只有美国自己

**Reply by lisongrui** [reply]
> 美国以此为荣:能打败美国的只有美国自己

除非掀桌子都不活了 大国堡垒都是从内部攻破的

**Comment by zhaokai5520** [comment]
政府把所有的资源都绑架了房子，刚需买房有什么错？

**Reply by imhappyok** [reply]
错在你没有躺平

**Comment by translace** [comment]
饵咸钩直

**Comment by ucanme** [comment]
抛开事实不谈，住在桥洞谈社会主义终极目标？还是谈共产主义的意识形态？还是谈资本主义的万恶？

**Reply by 306732407** [reply]
桥洞地面已经改成三角锥，住桥洞不可能的，多影响市容市貌啊【狗头保命】

**Comment by wdengf88** [comment]
我该死........................

**Reply by liuwenwu520** [reply]
别这样，我觉得还是银行、ZF和开发商占主要错误，特别是跑路的开发商

**Comment by Stevenphil** [comment]
坐等全国性金融风暴

**Comment by hash070** [comment]
![image](https://user-images.githubusercontent.com/62081013/178853802-69613d13-74da-49e6-bbbc-2c6ced784d20.png)

**Comment by ddh643** [comment]
利益分配出现了大问题，少部分人赚了大钱。

**Comment by z-huabao** [comment]
把知乎的不良风气带到了这里——阴阳怪气

**Reply by 0xhelium** [reply]
在知乎的话，阴阳怪气不算不良风气。 直接说的话，号没了找你吗？

**Reply by A359003** [reply]
名场面：“你已经在GitHub了，想说什么直接说，不用拐弯抹角”

**Reply by pengyidong** [reply]
要是可以直接说，干嘛还要阴阳怪气

**Comment by zhb45665** [comment]
虎兕出于柙,龟玉毁于椟中,是谁之过与？

**Reply by zhaomingtao-fighting** [reply]
和珅知道

**Comment by hkcs2260** [comment]
买房的还真没有什么错！！这条受害者有罪论的最大理由是“没人逼你买房”，可事实上却是所有的一切都在逼你买房！！户口、婚姻、生育、医疗、教育哪个不跟房子挂钩？最重要的是房地产行情毁掉了太多人的规划和理想、毁掉了太多人的价值观！！对老板来说实体经济投两个亿赚一千万没准还会赔本，房地产投两个亿赚四个亿，你做哪个生意？做实体的意义何在？对个人来说今年存的钱能换10个平方，再奋斗一年，两年存的钱还能换8个平方，然后三年存的钱还能换5个平方…工作的意义又何在？一次次调控，一次次翻倍,成功的掰弯了XX的公信力，也掰弯了全社会的价值观！做什么生意都不去做房地产，干什么工作都不如买房炒房！买不了也要砸锅卖铁去买，反正你干啥也特么不如房子涨的快！

**Reply by spider1211368975** [reply]
个体都是被利益集团裹挟到这个游戏里的，那些炒房投机的除外。如果业主有错，错也是最小的。

**Reply by ghost** [reply]
楼主开玩笑而已……

**Reply by A359003** [reply]
XX是什么？这是GitHub，你可以随便说

**Comment by jokerxsun** [comment]
快说，你是那个互联网公司里面的

**Comment by fkhisd** [comment]
买东西有什么错，认涨认跌，问题是现在钱交了，不给东西。

**Comment by mozhengxin** [comment]
Github上不了头条 灭了吧

**Comment by getinlight** [comment]
问题是监管允许卖房呀。钱付了，东西没拿到，责任在卖房者吗？

**Comment by Myst1615731035** [comment]
你去娶老婆，女方说要彩礼，你借钱把彩礼出了，借钱的人和女方认识，然后借钱的人和女方一起和你沟通说，你和女方结婚，然后你慢慢还钱，然后等了几年，女方还没准备好和你结婚，你依旧在慢慢还钱，结果女方跑了，借钱的人还要找你要钱。
TMD，和老子玩仙人跳呢？

**Comment by guocaoyi** [comment]
抛的挺好的，下次不许这么抛了

**Comment by yonee77** [comment]
列举下，买房十宗罪

**Comment by Jesse3692** [comment]
![微信图片_20220711141238](https://user-images.githubusercontent.com/24291671/178918790-1aa3b9dd-6ecd-4c9c-a46c-cbc34b683987.jpg)

**Comment by hqqlucky** [comment]
错就在没有用美国的石头砸脚

**Comment by kiss5523** [comment]
明天来办理一下离职手续

**Comment by kottory** [comment]
墙外平台不需要搞墙内狗头这一套

**Comment by BobbyYang9** [comment]
![微信图片_20220714155039](https://user-images.githubusercontent.com/74391151/178930758-35ff9c15-9e79-48d0-9384-7878efecbf07.jpg)

**Comment by muamuamu** [comment]
抛开事实不谈，哈哈哈哈，今天最佳笑话😂

**Comment by dearfuture** [comment]
gaka: 连停贷的后果都不怕，这已经不是一般的买房客了，这明显全都是美分

**Comment by liaopen123** [comment]
哈哈哈 楼主你真会钓鱼啊 。。。

**Comment by kliyes** [comment]
今天在某程序员群里还真有人说：“谁叫你们买期房的，活该，出了问题就闹事”，我真想顺着网线去揍他一顿。

**Comment by endlesspoi** [comment]
不要搬起石头砸自己的脚(怒｀Д´怒)

**Comment by dawnFlyme** [comment]
买个东西还他么一手交钱一手交货

**Comment by xiaohao587** [comment]
和房不房没有关系，即使没有房子一样会在其他地方收拾你，你存款，你孩子，你红码，你吃饭，你还以为你能逃过维尼的手掌心？

**Comment by Jony-Sminth** [comment]
这里讲这个又不会捂嘴封号，直接说吧，唉...

**Comment by chenkan0617chris** [comment]
不是，你不会是在学zard说话吧

**Comment by weimo123** [comment]
你妈的个狗屁

**Comment by pengyidong** [comment]
你是华东小偷大学毕业的吧

**Comment by volans19** [comment]
没错啊，一个愿打一个愿挨，一个愿意被割一个割的乐此不疲

**Comment by cjgfor327** [comment]
坐等二十大

**Comment by seeker-yao** [comment]
你的信件已经收到，o(∩_∩)o...哈哈

**Comment by hclgdb** [comment]
抛开事实不谈，为什么不把多余的房子租出去，多余的车去开滴滴来还房贷呢。

**Comment by seeker-yao** [comment]
你的信件已经收到，o(∩_∩)o...哈哈

**Comment by seeker-yao** [comment]
你的信件已经收到，o(∩_∩)o...哈哈

**Comment by ucanme** [comment]
王八蛋政腐没办法



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2025年2月17日(周一) 下午5:46
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [WeNeedHome/SummaryOfLoanSuspension] 抛开事实不谈，你们这群买房的难道一点错都没有吗？ (Discussion #123)




 
我承认我买期房确实没考虑清楚，银行贷款我会尽力去还的，维护我的信誉，承担我的责任！
 那么卖我房子的那个销售，她欺骗我房子已经建设十多层了，三年内能建设好，她的信誉，她的责任呢？
 恒大与汕头市恒明房地产开发有限公司开发商预期8个月未交房，他们的信誉，他们的责任？
 银行选择与多栋楼盘未交付的开发商合作，未封顶就放贷给开发商，他们的信誉，他们的责任？
 相关管理部门发放五证给多栋楼盘未交付的开发商售卖房子以及出现问题的当年业主就反馈相关部门，回复工程建设时间充足，他们的信誉，他们的责任？

**Comment by chinamickey** [comment]
规则在强者这里无效，在弱者这里就生效了。

缺乏契约精神是当前的通病；



***@***.***
 
发件人： MJcc1112
发送时间： 2025-02-17 17:46
收件人： WeNeedHome/SummaryOfLoanSuspension
抄送： Subscribed
主题： Re: [WeNeedHome/SummaryOfLoanSuspension] 抛开事实不谈，你们这群买房的难道一点错都没有吗？ (Discussion #123)
我承认我买期房确实没考虑清楚，银行贷款我会尽力去还的，维护我的信誉，承担我的责任！
那么卖我房子的那个销售，她欺骗我房子已经建设十多层了，三年内能建设好，她的信誉，她的责任呢？
恒大与汕头市恒明房地产开发有限公司开发商预期8个月未交房，他们的信誉，他们的责任？
银行选择与多栋楼盘未交付的开发商合作，未封顶就放贷给开发商，他们的信誉，他们的责任？
相关管理部门发放五证给多栋楼盘未交付的开发商售卖房子以及出现问题的当年业主就反馈相关部门，回复工程建设时间充足，他们的信誉，他们的责任？
